src/models/app_data_json_format.rs

```diff
@@ -8,18 +8,23 @@ use serde_with::serde_as;
 #[derive(Eq, PartialEq, Clone, Debug, Deserialize, Serialize, Hash, Default)]
 pub struct Referrer {
     pub address: H160,
+    pub kind: String,
```

Comment:
This is no longer on the official documentation. We don't use it.

It was just in the test data, to see whether we would leave out other unrelated fields.

Reply:
Cool will remove.